pinoSeqStream.js

```diff
@@ -96,6 +100,30 @@ class PinoSeqStream extends stream.Writable {
     }
   }
 
+  // When flushing logs pino doesn't actually call _final and wait for the logs to flush when using streams
+  // Instead it calls a poorly documented flushSync method that is not part of the stream.Writable interface
+  // As the name suggests the function is not async and will block the event loop until the logs are flushed
+  // Normally this wouldn't be an issue, but when running inside a Cloud function the function will often exit before the logs are flushed,
+  // so this implementation is in line with what pino recommends https://getpino.io/#/docs/asynchronous?id=aws-lambda
+  flushSync() {
```

Comment:
The `.d.ts` file should also be updated to expose this method on the return type